src/libraries/System.Formats.Asn1/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
I thought you wrote on a comment that the whole pourpose of this is to create an OOB assembly but this property is making it so that it is also shipped via the shared framework. Is that intentional?

Reply:
It's temporarily intentional.  To quote the top item for the fourth commit:

> This commit pulls System.Formats.Asn1 into the shared framework. We can alter this by moving (back) to shared source, ilmerge, or other build-time trickery to remove the package dependency; but it can be done as a low-noise followup.

Reply:
Ok, if intentional there is nothing wrong as we have this with other assemblies that are also half in half out but it could cause problems whenever we service the package in the future as we would want to ensure that this implementation would be used instead of the one shipped inbox everywhere, specially if your assembly contains exchange types.